tests/Public/New-GuestConfigurationPolicy.Tests.ps1

```diff
@@ -690,6 +690,75 @@ Describe 'New-GuestConfigurationPolicy' {
                         Assert-GuestConfigurationPolicyDefinitionFileValid @assertionParameters
                     }
                 }
+
+                Context 'Optional identity parameters - ManagedIdentityResourceId and LocalContentPath with ExcludeArcMachines' {
+                    BeforeAll {
+                        $fileName = $ContentUri.Split('/')[-1]
+                        $filePath = Join-Path -Path $defaultDefinitionsPath -ChildPath $fileName
+                        Invoke-WebRequest $ContentUri -OutFile $filePath
+
+                        $basePolicyParameters['ManagedIdentityResourceId'] = 'myManagedIdentity'
+                        $basePolicyParameters['LocalContentPath'] = $filePath
+                        $baseAssertionParameters['ExpectedManagedIdentity'] = 'myManagedIdentity'
+                    }
+
+                    It 'Should include contentManagedIdentity in the result object' {
```

Comment:
Can you check that ExcludeArcMachines is working properly and is excluding Arc?

Reply:
Added a check for ExcludeArcMachines.